packages/api/api-generator/src/commands/generate/generate-command.ts

```diff
@@ -8,32 +8,37 @@ import { generateCrudSingle } from "./generateCrudSingle/generate-crud-single";
 import { writeGeneratedFiles } from "./utils/write-generated-files";
 
 export const generateCommand = new Command("generate").action(async (options) => {
-    const orm = await CLIHelper.getORM(undefined, undefined, { dbName: "generator" });
+    try {
```

Comment:
As @nsams  noted, I refactored the try-catch mechanism to be more locally, but honestly, it just makes everything more complicated - I would recommend to have the try catch across the full function.

Reply:
not really complicated imho